src/Lykke.Sdk/Health/HealthIssuesCollection.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using JetBrains.Annotations;
+
+namespace Lykke.Sdk.Health
+{
+    [PublicAPI]
+    public class HealthIssuesCollection : IReadOnlyCollection<HealthIssue>
```

Comment:
This already exist in Lykke.Common - https://github.com/LykkeCity/CommonDotNetLibraries/blob/master/src/Lykke.Common/Health/HealthIssuesCollection.cs